homeassistant/components/lifx/sensor.py

```diff
@@ -29,6 +30,14 @@
     entity_registry_enabled_default=False,
 )
 
+ZONES_SENSOR = SensorEntityDescription(
+    key=ATTR_ZONES,
+    name="Zones",
+    entity_category=EntityCategory.DIAGNOSTIC,
+    state_class=SensorStateClass.MEASUREMENT,
+    entity_registry_enabled_default=True,
```

Comment:
Should this be false?

Reply:
I thought about this, but I figured the sensor is pretty pertinent to multizone devices, in that it's the only way to know either the number of zones or the state of each zone. So I reckon it should be enabled by default.

Reply:
I thought about this one some more. I think its ok to do as sensors if we disable them by default otherwise we have a scale problem since things start to fall over once we hit 4096 entities

Reply:
If the answer to the problem is, "we'll do it the usual way, because no-one will bother actually using it/turning it on" then somethingh is very wrong. You can implement them as individual sensors if you like, but I don't feel like making the user experience actively hostile.